samples/VirtualSchemes.Basic/README.md

```diff
@@ -0,0 +1,12 @@
+AuthSamples.VirtualScheme.Basic
```

Comment:
Should this be named VirtualScheme.DefaultSelector?

Reply:
Yeah I'm still trying to figure out whether to cram more 'basic' flows into this sample or create multiple VirtualScheme samples.